acsf-site-setup-playbook.yml

```diff
@@ -39,4 +39,5 @@
     - { role: setup-local }
     - { role: setup-site }
     - { role: add-vhost }
-  serial: 12
+  serial: 6
+  max_fail_percentage: 100
```

Comment:
![wcgw](https://freedomandheritage.org.au/wp-content/uploads/2018/07/not_business_intelligence.jpg)

Reply:
Funny you should ask that, because I _do_ know the answer to that question 😈

Reply:
(_Narrator: they all fail, and the playbook stops._)